orleans/JournaledTodoList/JournaledTodoList.WebApp/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿using JournaledTodoList.WebApp.Components;
+using JournaledTodoList.WebApp.Services;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.AddServiceDefaults();
+builder.AddKeyedAzureTableClient("clustering");
+builder.AddKeyedAzureBlobClient("grain-state");
+builder.UseOrleans(siloBuilder =>
+{
+    siloBuilder.AddLogStorageBasedLogConsistencyProviderAsDefault();
+    siloBuilder.AddStateStorageBasedLogConsistencyProvider(name: Constants.StateStorageProviderName);
+});
```

Comment:
With this set up, the log consistency providers are using the grain state as storage, in this case, blob storage. When the log-based providers save state, does it just save the added/raised event(s), or does it save all in one go? That is, if there are 1000 events, will they all be serialized and save to blob storage or just the new events?